frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -227,14 +227,14 @@ string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-parent-status-tool
 
 string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text.ahead-of-remote=(ahead of {0})
 string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text.behind-remote=(behind {0})
-#To avoid clutter we omit `& newer than` part in status label, coz this is default situation
+# To avoid clutter we omit `& newer than` part in status label, coz this is default situation
 string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text.diverged-from-and-newer-than-remote=(diverged from {0})
 string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text.diverged-from-and-older-than-remote=(diverged from & older than {0})
 string.GitMachete.BranchOrCommitCellRendererComponent.sync-to-remote-status-text.untracked=(untracked)
 
 string.GitMachete.EnhancedGraphTable.automatic-discover.notification.title.cannot-discover-layout-error=Can't automatically discover repository layout
 string.GitMachete.EnhancedGraphTable.automatic-discover.task-title=Automatically discovering branch layout
-string.GitMachete.EnhancedGraphTable.automatic-discover.success-message=Branch layout has been automatically discovered
+string.GitMachete.EnhancedGraphTable.automatic-discover.success-message=Branch layout has been automatically discovered. Consider using ''{0}'' button if the layout needs to be manually patched
```

Comment:
TBD if that's the formulation of this message... but from what I've seen from exp of users, this might be useful to communicate

Reply:
note that it is possible to add link action to the notification

Reply:
Right, that would be nice... actually, do you know/remember from the top of your head how to do it? I can find myself ofc if not ;)

Reply:
I remember it from there: `git4idea.branch.GitDeleteBranchOperation#notifySuccess`
The notification above shows after git4idea branch deletion and gives a possibility to restore the deleted branch.

Smth similar should be achievable

Reply:
Ok, done:

![image](https://user-images.githubusercontent.com/3383210/103218665-f3e8d880-491b-11eb-9cd4-fbb0104f8336.png)